rental.rb

```diff
@@ -0,0 +1,13 @@
+require_relative('./book')
+require_relative('./person')
+require_relative('./student')
+
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, _book, _person)
+    @date = date
+    @book.rental << self
+    @person.rentals << self
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think you should define `@book` and `@person` properly to avoid errors.

  ```
  @book = book
  book.rentals << self

  @person = person
  person.rentals << self
  ```